haystack/utils/docker.py

```diff
@@ -19,12 +26,6 @@ def cache_models(models: Optional[List[str]] = None, use_auth_token: Optional[Un
     if models is None:
         models = ["deepset/roberta-base-squad2"]
 
-    # download punkt tokenizer
-    logging.info("Caching punkt data")
-    import nltk
-
-    nltk.download("punkt")
```

Comment:
Should we mark the PR breaking? Custom docker images calling `cache_models` won't cache nltk anymore - which is what we want by design, but maybe worth it calling out in the release notes?

Reply:
Absolutely :100: